pydicom/config.py

```diff
@@ -71,6 +72,69 @@ def DS_decimal(use_Decimal_boolean=True):
 logger.addHandler(handler)
 
 
+image_handlers = []
+"""Image handlers for converting pixel data.
```

Comment:
you need to connect, then go in artifact /home/ubuntu/doc/_build/html/index.html

Something like that. The problem is that the docstring here will not be render anywhere.